modules/content.js

```diff
@@ -0,0 +1,135 @@
+// select the items
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const displayArea = document.querySelector('#showBooks');
+const list = document.querySelector('#booksshow');
+const addNew = document.querySelector('#new');
+const contact = document.querySelector('#contact');
+const addBook = document.querySelector('#adding');
+const contactInfo = document.querySelector('#reachme');
+
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  storeData() {
+    const bookTitle = title.value;
+    const bookAuthor = author.value;
+    const book = new Books(bookTitle, bookAuthor);
+    // Check if local storage is empty
+    if (localStorage.getItem('books') === null) {
+      const bookShelf = [];
+      bookShelf.push(book);
+      localStorage.setItem('books', JSON.stringify(bookShelf));
+    } else {
+      const bookShelfstr = localStorage.getItem('books');
+      const bookArray = JSON.parse(bookShelfstr);
+      bookArray.push(book);
+      localStorage.setItem('books', JSON.stringify(bookArray));
+    }
+    // Clear input
+    title.value = '';
+    author.value = '';
+    this.displayBooks();
+  }
+
+  displayBooks() {
+    const wrapper = document.createElement('div');
+    const line = document.createElement('hr');
+    const bookShelfstr = localStorage.getItem('books');
+    const tits = document.createElement('div');
+    tits.innerText = this.author;
+    const bookArray = JSON.parse(bookShelfstr);
+    if (bookArray && bookArray.length) {
+      bookArray.forEach((element, index) => {
+        const displayTitle = document.createElement('p');
+        const displayAuth = document.createElement('p');
+        const deletebtn = document.createElement('div');
+        const container = document.createElement('div');
+        const words = document.createElement('div');
+        // set attributes
+        displayTitle.innerText = `"${element.title}" by`;
+        displayAuth.innerText = element.author;
+        deletebtn.innerHTML = `<button class="btn borders" onclick='deleteItem(${index})'>Remove</button>`;
```

Comment:
- [ ] When I click to delete a book the book is not removed from the local storage or the page and I get this error. Kindly fix that.

  ![fkffo](https://user-images.githubusercontent.com/54848363/205715842-134823da-da7b-4c2a-906b-4dfb195bfcd9.jpg)